source/marc/enums.ttl

```diff
@@ -812,58 +812,63 @@
 
 :LessThanFullLevelMaterialNotExamined a :EncLevelType ;
     #owl:sameAs :EncLevelType-2 ;
-    rdfs:comment "Retrospektiv inmatning av reducerade poster, såsom ur en kortkatalog Sökelement kan avspegla äldre praxis."@sv;
+    rdfs:comment "Retrospektiv inmatning av reducerade poster, såsom ur en kortkatalog. Sökelement kan avspegla äldre praxis."@sv;
     skos:notation "2" ;
-    skos:prefLabel "Less-than-full level, material not examined"@en, "Retrospektiv nivå: enligt katalogkort e.d."@sv .
+    skos:prefLabel "Less-than-full level, material not examined"@en, "Retrospektiv: enligt katalogkort e.d."@sv .
 
 :AbbreviatedLevel a :EncLevelType ;
     #owl:sameAs :EncLevelType-3 ;
-    rdfs:comment "jfr KRS 1.0D1"@sv;
+    rdfs:comment "Miniminivå med kontrollerade sökelement."@sv;
+    skos:note "Motsvarar KRS 1.0D beskrivningsnivå 1, den minst fullständiga."@sv;
     skos:relatedMatch <http://id.loc.gov/vocabulary/menclvl/3>;
     skos:notation "3" ;
     skos:prefLabel "Abbreviated level"@en, "Miniminivå"@sv .
 
 :CoreLevel a :EncLevelType ;
     #owl:sameAs :EncLevelType-4 ;
-    rdfs:comment "jfr KRS 1.0D, anm 1"@sv;
+    kbv:category kbv:pending ;
+    rdfs:comment "Förkortad katalogisering, främst på verknivå."@sv;
     skos:relatedMatch <http://id.loc.gov/vocabulary/menclvl/4>;
     skos:notation "4" ;
-    skos:prefLabel "Core level"@en, "Verknivå (förkortad katalogisering)"@sv .
+    skos:prefLabel "Core level"@en, "Verknivå (förkortad)"@sv .
 
 :PartialPreliminaryLevel a :EncLevelType ;
     #owl:sameAs :EncLevelType-5 ;
-    rdfs:comment "Hit till exempel poster som är baserade på förvärvsinformation eller på annat sätt okontrollerade."@sv;
+    rdfs:comment "Hit till exempel beställningsposter baserade på förvärvsinformation, eller på annat sätt okontrollerade."@sv;
     skos:relatedMatch <http://id.loc.gov/vocabulary/menclvl/5>;
     skos:notation "5" ;
-    skos:prefLabel "Partial (preliminary) level"@en, "Preliminär post (hit beställningsposter)"@sv .
+    skos:prefLabel "Partial (preliminary) level"@en, "Preliminär"@sv .
 
 :MinimalLevel a :EncLevelType ;
     #owl:sameAs :EncLevelType-7 ;
-    rdfs:comment "jfr KRS 1.0D2"@sv;
+    rdfs:comment "Normalvärde för Libris-bibliotekens katalogisering."@sv;
+    skos:note "Motsvarar KRS 1.0D beskrivningsnivå 2, den av biblioteken normalt tillämpade."@sv;
     skos:relatedMatch <http://id.loc.gov/vocabulary/menclvl/7>;
     skos:notation "7" ;
     skos:prefLabel "Minimal level"@en, "Biblioteksnivå"@sv .
 
 :PrepublicationLevel a :EncLevelType ;
     #owl:sameAs :EncLevelType-8 ;
-    rdfs:comment "Post upprättad innan objektet givits ut"@sv;
+    rdfs:comment "Post upprättad innan resursen givits ut. Hit även CIP-poster."@sv;
     skos:relatedMatch <http://id.loc.gov/vocabulary/menclvl/8>;
     skos:notation "8" ;
-    skos:prefLabel "Prepublication level"@en, "CIP-post"@sv .
+    skos:prefLabel "Prepublication level"@en, "Förhandsinformation"@sv .
```

Comment:
Have everyone agreed to change prefLabel to "Förhandsinformation", even if we don't supplement with hovering info (that it also applies to CIP records)?

Reply:
No feedback so far, let's merge so it is visible in our dev environments for evalutation.